File: CSC1700/Functions/rectangle.py
def area(width, length):
    return width * length

def perimeter(width, length):
    return 2 * (width + length)

def main():
    my_perimeter = perimeter(5, 6)
    print(f"In {__name__}, perimeter is: {my_perimeter}")

print(f"The module name is {__name__}")
if __name__ == "__main__":
    main()

File: CYB 2810 GitHub/Rectangle.java
public class Rectangle {
    double length;
    double width;

    double area() {
        return length * width;
    }

    double perimeter() {
        return length * 2 + width * 2;
    }

    public static void main(String[] args) {
        Rectangle rectangle = new Rectangle();
        rectangle.length = 5.0;
        rectangle.width = 3.0;

        System.out.println("The area is " + rectangle.area());
        System.out.println("The perimeter is " + rectangle.perimeter());
    }
}
